src/pages/Post.js

```diff
@@ -517,6 +522,39 @@ class PostPage extends React.Component {
       endDate: endDate
     })
   }
+  
+  showFilters() {
+    if (this.state.filterToggle == false) {
+      document.getElementById("yearBoxes").style.display = "block"
+      document.getElementById("schoolBoxes").style.display = "block"
+      this.setState({
+        filterToggle: true,
+        filterToggleText: "Remove Filters",
+        filterToggleColor: "#a32512"
+      })
+    } else {
+      document.getElementById("yearBoxes").style.display = "none"
+      document.getElementById("schoolBoxes").style.display = "none"
+      this.setState({
+        filterToggle: false,
+        filterToggleText: "Add Filters",
+        filterToggleColor: "#12a340"
+      })
```

Comment:
Good but we will want to refactor this. First, it's generally not good practice in React to use ```document.getElementById```. Instead, we should be using React refs. Here is documention on that: https://reactjs.org/docs/refs-and-the-dom.html. It is also not necessary to set state for the text or the color. Instead, we should check ```filterToggle``` in ```render()``` and set the text and color accordingly.

Reply:
See #9